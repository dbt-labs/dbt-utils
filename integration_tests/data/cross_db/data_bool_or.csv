key,val1,val2
abc,1,1
abc,1,0
def,1,0
hij,1,1
hij,1,
klm,1,0
klm,1,