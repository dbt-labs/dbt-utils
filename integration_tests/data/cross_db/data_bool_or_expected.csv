key,value
abc,true
def,false
hij,true
klm,false